src/mcp_agent/workflows/llm/multipart_converter_anthropic.py

```
@@ -0,0 +1,518 @@
+        text = get_text(resource)
+        if text:
+            return TextBlockParam(type="text", text=text)
+
+        # This is for binary resources - match the format expected by the test
+        if isinstance(resource.resource, BlobResourceContents) and hasattr(
+            resource.resource, "blob"
+        ):
+            blob_length = len(resource.resource.blob)
+            return TextBlockParam(
+                type="text",
+                text=f"Embedded Resource {uri._url} with unsupported format {mime_type} ({blob_length} characters)",
```

Pylint (3.3.7)

[convention] Line too long (116/100)

(C0301)